examples/fused_gat.py

```diff
@@ -0,0 +1,88 @@
+import argparse
```

Comment:
Maybe we can add a `benchmark/fuse` repo instead and compare runtimes between `GATConv` and `FusedGATConv`?